### DISCUSSION TITLE
Deprecation of altool

### DISCUSSION THREAD

**Discussion by borjafdezgauna** [picoe/Eto#2500] [General]
Hi there,
I just got an email from Apple saying that notarization via altool will stop working the 1st of November. Any plans to change this? Should I try to do it myself and send a pull request?
Thanks!
Borja

**Comment by cwensley** [comment]
Hey @borjafdezgauna, thanks for bringing this up!  I didn't even know that altool was deprecated. Looking at the updated [Customizing the notarization workflow](https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution/customizing_the_notarization_workflow?language=objc) docs, this should be relatively easy to update.  We can rip out the polling stuff I did too and just pass `--wait` to the command instead.  I've created issue #2502 to track getting that migrated over.
